### DISCUSSION TITLE
Function Descriptions for library functions.

### DISCUSSION THREAD

**Discussion by mikekenn** [microsoft/vscode-cpptools#8529] [Q&A] [answered]
Greetings,

I'm wondering if there is a way to enable function Descriptions? I can generate them for custom made classes/functions using doxygen syntax ... But nothing seems to be appearing for functions available in say the vector library?

I'm running this on macOS, latest version.

@sean-mcmanus 

I see you commented in the following [discussion](https://github.com/microsoft/vscode-cpptools/discussions/8428) that you only parse the header files for comments, is this also true for c++?

I am not getting any function information returned for any standard library calls.

macOS version: Monterey 12.0.1
VScode version: 1.63.0 (Universal)
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Commit: 7db1a2b88f7557e0a43fec75b6ba7e50b3e9f77e
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Date: 2021-12-07T05:15:48.091Z
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Electron: 13.5.2
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Chromium: 91.0.4472.164
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Node.js: 14.16.0
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; V8: 9.1.269.39-electron.0
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; OS: Darwin x64 21.1.0
C/C++ Extension version: v1.8.0-insiders2

Thanks!

**Comment by sean-mcmanus** [comment, marked as answer]
Yes, the headers themselves have to be commented.
